in_toto/util_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux || darwin || !windows
 // +build linux darwin !windows
```

Comment:
According to https://pkg.go.dev/go/build/constraint@go1.16.7, the parser for 1.16 also supports the new go:build constraints. Can we check this out?

Reply:
The tests succeed, so I guess this works. The CI just fails because of network issues with coverall..

Reply:
Oh yes, I'm just wondering if we can get rid of ` // +build linux darwin !windows ` as even 1.16 seems to support the `go:build` format.

Reply:
Ah okay, now I get what you mean... I think this would make our code incompatible with Go < 1.16. I would argue we wait a little bit more.. this string/comment has no effect on future Go versions and we are not losing anything by adding it.

Reply:
Fair enough.